packages/server/src/api/controllers/row/index.ts

```diff
@@ -198,8 +198,21 @@ export async function destroy(ctx: UserCtx<DeleteRowRequest>) {
 export async function search(ctx: Ctx<SearchRowRequest, SearchRowResponse>) {
   const tableId = utils.getTableId(ctx)
 
+  // Current user context for bindable search
+  const { _id, _rev, firstName, lastName, email, status, roleId } = ctx.user
```

Comment:
Can this be a something that lives in the user SDK? Under `server/src/sdk/users/utils.ts` there are some utilities, a function like `getUserContextBindings(ctx.user)` would help describe what is happening here, and would mean you can just use it like:
```ts
const enrichedQuery = await utils.enrichSearchContext(
    { ...ctx.request.body.query },
    {
      user: sdk.users.getUserContextBindings(ctx.user),
    }
  )
```

Reply:
Yep that makes perfect sense. I'll get it shifted out 👍